File: calculosalarial.js
// Solicita ao usuário os valores
const valorSalario = parseFloat(prompt("Digite o valor do salário:"));
const valorBeneficio = parseFloat(prompt("Digite o valor dos benefícios:"));

// Calcula o imposto
const valorImposto = calcularImposto(valorSalario);

// Cálculo do salário final
const saida = valorSalario - valorImposto + valorBeneficio;

// Exibe o resultado formatado
console.log(`O valor final é: ${saida.toFixed(2)}`);

function calcularImposto(salario) {
  let aliquota = 0;

  if (salario >= 0 && salario <= 1100) {
    aliquota = 0.05;
  } else if (salario > 1100 && salario <= 2500) { 
    aliquota = 0.10;
  } else {
    aliquota = 0.15;
  }

  return aliquota * salario;
}

File: CalculoSalarial.cs
using System;

public class Desafio{

    public static void Main()
    {
        Console.WriteLine("Digite o valor do salário:");
        float valorSalario = float.Parse(Console.ReadLine());
        Console.WriteLine("Digite o valor dos benefícios:");
         float valorBeneficio = float.Parse(Console.ReadLine());

        float valorImposto = 0;
        if (valorSalario>=0 && valorSalario<=1100){
            valorImposto = 0.05F * valorSalario;
        
        }
          else if (valorSalario>=1100.01 && valorSalario<=2500.00){

            valorImposto=0.10F *valorSalario;
        }

         else{

            valorImposto=0.15F *valorSalario;
        }
        float saida = valorSalario - valorImposto+ valorBeneficio;
        Console.WriteLine(saida.ToString("0.00"));
    }
    
}